Assets/MixedRealityToolkit-SDK/Features/Audio/Influencers/AudioLoFiEffect.cs

```diff
@@ -0,0 +1,208 @@
+                AudioLoFiSourceQualityType.AmRadio,
+                new AudioLoFiFilterSettings(40, 5000));     // Frequency range: 40 Hz - 5 kHz
+            sourceQualityFilterSettings.Add(
+                AudioLoFiSourceQualityType.FmRadio,
+                new AudioLoFiFilterSettings(30, 15000));    // Frequency range: 30 Hz - 15 kHz
+        }
+
+        /// <summary>
+        /// Settings for the filters used to simulate a low fidelity sound source.
+        /// </summary>
+        /// <remarks>
+        /// This struct is soley for the private use of the AudioLoFiEffect class.
+        /// </remarks>
+        private struct AudioLoFiFilterSettings
+        {
+            /// <summary>
+            /// The frequency below which sound will be heard.
+            /// </summary>
+            public float LowPassCutoff
+            { get; private set; }
+
+            /// <summary>
+            /// The frequency above which sound will be heard.
+            /// </summary>
+            public float HighPassCutoff
+            { get; private set; }
+
+            /// <summary>
+            /// FilterSettings constructor.
+            /// </summary>
+            /// <param name="highPassCutoff">High pass filter cutoff frequency.</param>
+            /// <param name="lowPassCutoff">Low pass filter cutoff frequency.</param>
+            public AudioLoFiFilterSettings(float highPassCutoff, float lowPassCutoff) : this()
+            {
+                HighPassCutoff = highPassCutoff;
+                LowPassCutoff = lowPassCutoff;
+            }
+
+            /// <summary>
+            /// Checks to see if two FilterSettings objects are equivalent.
+            /// </summary>
+            /// <returns>True if equivalent, false otherwise.</returns>
+            public static bool operator ==(AudioLoFiFilterSettings a, AudioLoFiFilterSettings b)
+            {
+                return a.Equals(b);
+            }
+
+            /// <summary>
+            /// Checks to see if two FilterSettings objects are not equivalent.
+            /// </summary>
+            /// <returns>False if equivalent, true otherwise.</returns>
+            public static bool operator !=(AudioLoFiFilterSettings a, AudioLoFiFilterSettings b)
+            {
+                return !(a.Equals(b));
+            }
+
+            /// <summary>
+            /// Checks to see if a object is equivalent to this FilterSettings.
+            /// </summary>
+            /// <returns>True if equivalent, false otherwise.</returns>
+            public override bool Equals(object obj)
+            {
+                if (obj == null)
+                {
+                    return false;
+                }
+
+                if (!(obj is AudioLoFiFilterSettings))
+                {
+                    return false;
+                }
+
+                AudioLoFiFilterSettings other = (AudioLoFiFilterSettings)obj;
+                if ((other.LowPassCutoff != LowPassCutoff) ||
```

Comment:
`!other.LowPassCutoff.Equals(LowPassCutoff)` so we get proper floating point checks. (same for the line below)